starboard/tools/abstract_launcher.py

```diff
@@ -95,8 +96,42 @@ def LauncherFactory(platform_name,
       **kwargs)
 
 
+class ReturnCodeStatus(IntEnum):
+  """Represents status of code, returned by target"""
+
+  #: Target exited normally. Return code is available
+  OK = 0
+
+  #: Target exited normally. Return code is not available
+  NA = 1
+
+  #: Target crashed
```

Comment:
What will CRASH and ISSUE be used for? If they are used to match some standards defined elsewhere, maybe it's worth adding a link to those here?

Reply:
These constants where suggested by @kaidokert in the issue description: [b/286272327](https://partnerissuetracker.corp.google.com/issues/286272327). I'm not sure about if it matches any standards.

Reply:
Please, review the [changes in test_runner.py](https://github.com/youtube/cobalt/pull/672/commits/fcb028a4d18fa16ea5ce446c177229965aac002a)